src/main/java/org/apache/commons/compress/archivers/sevenz/SevenZFile.java

```diff
@@ -1090,7 +1108,13 @@ private void readFilesInfo(final ByteBuffer header, final Archive archive) throw
                 ++emptyFileCounter;
             }
         }
-        archive.files = files;
+        List<SevenZArchiveEntry> entries = new ArrayList<>();
```

Comment:
Maybe a final here?
```
final List<SevenZArchiveEntry> entries = new ArrayList<>();
```

Reply:
+1
I use `final` as much as possible to make it obvious when reading or debugging whether I need to even think about a given local variable, for more details on my reasoning, please see https://garygregory.wordpress.com/2013/01/26/the-final-kiss-in-java/